src/components/user/repo/registrationPin.repo.js

```diff
@@ -1,33 +1,27 @@
 const { registrationPinModel } = require('../../../services/user/model');
-const { GeneralError } = require('../../../errors');
+const { deleteManyResult } = require('../../helper/repo.helper');
+const { validateRequired } = require('../../helper/validation.helper');
 
+/**
+ * Return pseudonyms for email
+ * @param email
+ */
 const getRegistrationPinsForUser = async (email) => {
+	validateRequired(email);
```

Comment:
beside required, i would expect email to be a string which is not empty (length>0)